### ISSUE TITLE
WCAG 2.1 Accessibility issue: Name of slider is not defined

### ISSUE THREAD

**Issue by robertmausser** [nachocinalli/adapt-sliderLottie#1]
Name of slider is not defined.

Name of slider should be properly defined.

https://app.screencast.com/zYkA70KeIse3P

https://www.w3.org/WAI/ARIA/apg/patterns/slider/

The element serving as the focusable slider control has role [slider](https://w3c.github.io/aria/#slider).
